odl/space/pspace.py

```diff
@@ -873,24 +873,42 @@ def show(self, title=None, indices=None, **kwargs):
             else:
                 indices = list(np.linspace(0, self.size - 1, 4, dtype=int))
         else:
-            if isinstance(indices, tuple):
+            if (isinstance(indices, tuple) or
+                    (isinstance(indices, list) and
+                     not all(isinstance(idx, Integral) for idx in indices))):
+                # Tuples or lists containing non-integers index by axis.
```

Comment:
No it doesn't. Lists that contain slice objects behave the same as tuples in Numpy indexing:
```py
>>> a = np.arange(12).reshape((3, 4))
>>> a[[0, 1, 2]]
array([[ 0,  1,  2,  3],
       [ 4,  5,  6,  7],
       [ 8,  9, 10, 11]])
>>> a[[0, slice(1, 3)]]
array([1, 2])
>>> a[(0, slice(1, 3))]
array([1, 2])
```